src/front-end/src/components/MentorCompletion.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+export default () => (
+  <div className="MentorCompletion">
+    <p>
+      Congradulations! You have successfully created your own account. Please
+      activate your account through the email we just sent and click
+      <a
+        className="loginlink"
+        href="http://localhost:3000/login
+      "
+      >
+        here
+      </a>
+      to login!

```

Comment:
Add a space after "click" and a space before "to login"

Reply:
I was like what you said but the eslint asked me to remove those spaces.